### DISCUSSION TITLE
Can figures use the lightbox feature?

### DISCUSSION THREAD

**Discussion by jackbaty** [Lednerb/bilberry-hugo-theme#413] [Ideas]
Hello,

I'm thinking of porting my existing Hugo content over the the billberry theme. So far, in my testing, it's been pretty straightforward. 

One thing I'm dealing with is that my blog uses a lot of photographs in content, and my current theme (LoveIt) implements a lightbox gallery for any images on the page. I don't need the gallery portion, but I would like to have a lightbox available. The problem is that most of my images are in `<figure>` tags because I like to use `<figcaption>` for captions. It seems there's no lightbox effect for images within Figures.

Is there anything I can to to enable bilberry's lightbox for images within figures? I'd prefer this to switching all of them to standard Markdown images.

Thanks!

Jack

**Comment by igor-baiborodine** [comment]
@jackbaty, could you please provide me with a link to any page on your current blog that contains a lightbox within a `<figure>` tag?

**Reply by jackbaty** [reply]
Hi, this post is an example, although I suppose that technically it's a gallery and not just a single lightbox. 

https://baty.net/2022/roll-078-leica-mp/

Thanks.

**Comment by igor-baiborodine** [comment]
With the release of [v3.10.0](https://github.com/Lednerb/bilberry-hugo-theme/releases/tag/v3.10.0), the lightbox feature is available for an image within the `<figure>` tag.

Please see this test article I created on the Bilberry Sandbox:
- https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/blob/master/content/article/test-image-modal-zoom-with-figure.md
- https://www.bilberry-sandbox.kiroule.com/article/test-image-modal-zoom-with-figure/

**Reply by jackbaty** [reply]
This seems to work perfectly, thank you!
